analyzeTest.ts

```diff
@@ -0,0 +1,55 @@
+import * as fs from 'fs';
+
+const pluginName = process.env.PLUGIN;
+
+const getStatsFilePath = () => `./plugins/${pluginName}/dist/stats.json`;
+
+const toMB = (value: number) => value / 1024 ** 2;
+
+const stringifyMB = (value: ReturnType<typeof toMB>): string =>
+  `${value.toFixed(2)} MB`;
```

Comment:
nits (for important):
```suggestion
const toMiB = (value: number) => value / 1024 ** 2;

const stringifyMiB = (value: ReturnType<typeof toMiB>): string =>
  `${value.toFixed(2)} MiB`;
```